src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1038,14 +1019,15 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 
 				if (e.dataTransfer) {
 					e.dataTransfer.effectAllowed = 'copyMove';
+					e.dataTransfer.setDragImage(tab, 0, 0); // top left corner of dragged tab set to cursor position to make room for drop-border feedback
```

Comment:
This leaves the drag image with a white border on left and right, would maybe not show it initially to fix

Reply:
Found another solution

Reply:
Which one?